src/urllib3/__init__.py

```diff
@@ -30,17 +30,19 @@
 except ImportError:
     pass
 else:
-    # fmt: off
-    if (
-        not ssl.OPENSSL_VERSION.startswith("OpenSSL ")
-        or ssl.OPENSSL_VERSION_INFO < (1, 1, 1)
-    ):  # Defensive:
+    if not ssl.OPENSSL_VERSION.startswith("OpenSSL "):  # Defensive:
```

Comment:
This allows things that are neither OpenSSL nor LibreSSL, which I think is OK but you can disagree.